src/test/java/org/broadinstitute/hellbender/tools/exome/segmentation/CopyRatioSegmenterUnitTest.java

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.hellbender.tools.exome.segmentation;
+
+import org.broadinstitute.hellbender.tools.exome.ModeledSegment;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+/**
+ * Created by davidben on 6/6/16.
+ */
+public class CopyRatioSegmenterUnitTest {
+    @Test
+    public void testSegmentation() {
+        final double[] trueWeights = new double[] {0.2, 0.5, 0.3};
+        final double[] trueLog2CopyRatios = new double[] {-2.0, 0.0, 1.7};
+        final double trueMemoryLength = 1e5;
+
+        final double trueStandardDeviation = 0.2;
+
+        CopyRatioHiddenMarkovModel trueModel = new CopyRatioHiddenMarkovModel(trueLog2CopyRatios, trueWeights,
+                trueMemoryLength, trueStandardDeviation);
+
+        // randomly set positions
+        final Random random = new Random(271);
+        final int chainLength = 10000;
+        final List<SimpleInterval> positions = new ArrayList<>();
+        int position = 1;
+        for (int n = 0; n < chainLength; n++) {
+            position += random.nextInt((int) (trueMemoryLength/4));
+            final SimpleInterval interval = new SimpleInterval("chr1", position, position);
```

Comment:
perhaps add a simple test with multiple chromosomes, each a single segment with the same state?  this would test that chromosomes are always segmented.